Food_Tracker_App/app/src/main/res/layout/recipe_ingredient_content.xml

```diff
@@ -20,15 +20,23 @@
         android:layout_height="40dp"
         android:layout_weight="1"
         android:padding = "10dp"
-        android:text="amount" />
+        android:text="amt" />
 
     <TextView
         android:id="@+id/Category"
         android:layout_width="wrap_content"
         android:layout_height="40dp"
-        android:layout_weight="1"
+        android:layout_weight="3"
         android:padding = "10dp"
         android:text="category" />
 
+    <Button
+        android:id="@+id/delete_Button"
+        android:layout_width="48dp"
+        android:layout_height="wrap_content"
+        android:layout_weight="1"
+        android:text="X"
+        android:focusable="false"/>
```

Comment:
If you have layout weight you should set the `layout_width` to 0dp to make sure that it takes that into account, same for the other two `TextView`s